### PR TITLE
Add toast command

### PR DESCRIPTION
**Pull request by mheguy**

### CODE REVIEW

**Comment by plague006** [comment]
Causes desyncs in MP. Will comment again when fixed.

**Comment by mheguy** [comment]
Works fine now.

**Comment by grilledham** [comment]
We could consider renaming 'toast-all' to just 'toast' to make it consistent with 'popup'. Or we change 'popup' to 'popup-all'

**Comment by mheguy** [comment]
Tested in SP and MP.

File: features/gui/toast.lua [hunk 1]
```diff
@@ -247,4 +252,40 @@ function Public.toast_all_players(duration, message)
...
+        allowed_by_server = true
+    },
+    function(args)
+        local player_name = args.player
+        local player = game.players[player_name]
+        if player then
+            Public.toast_player(player, 15, args.msg)
+            Utils.print_admins(format('%s sent a toast to %s', Utils.get_actor(), player_name))
+        else
+            Game.player_print({'common.fail_no_target'}, Color.yellow)
```
Comment (grilledham) [inline features/gui/toast.lua]: Currently if you accidentally put a space in the player name like so 
`/toast-player grilledha m some message`
You get 'Target not found'

I feel this message could be better. Maybe something like 'player <player_name> not found'

**Review by grilledham — APPROVED** [review]

File: features/gui/toast.lua (hunk 1 above)
Comment (mheguy) [reply features/gui/toast.lua]: You're right the message was weak sauce. I took it from another command without thought to improving it.

**Comment by mheguy** [comment]
After refining error: Ran command once properly and once with error, both acted as expected.

**Pull request merged**